include/swift/Runtime/Metadata.h

```diff
@@ -1455,8 +1449,7 @@ struct TargetClassMetadata : public TargetHeapMetadata<Runtime> {
   /// if this is an artificial subclass.  We currently provide no
   /// supported mechanism for making a non-artificial subclass
   /// dynamically.
-  ConstTargetFarRelativeDirectPointer<Runtime, TargetNominalTypeDescriptor,
-                                      /*nullable*/ true> Description;
+  const TargetNominalTypeDescriptor<Runtime> *Description;
```

Comment:
Absolute addresses in metadata still need to be templated in order to support out-of-process clients like lldb and remote mirrors. You probably need `ConstTargetMetadataPointer` instead of `const *` here.